src/style.css

```
@@ -1,3 +1,64 @@
 body {
   margin: 0;
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1200px;
```

Consider it will be better
```suggestion
  width: 100%;
```

---

>Якщо я вказую width: 100%; замість max-width: 100%; тоді не проходить тест 1 та 2.